tasks/custom_task/custom_task.py

```diff
@@ -14,10 +14,33 @@
 
 from invoke import Context
 
-DD_INVOKE_LOGS_PATH = "/tmp/dd_invoke.log"
+from tasks.libs.common.color import color_message
 
 
-def log_invoke_task(name: str, module: str, task_datetime: str, duration: float, task_result: str) -> None:
+def get_dd_invoke_logs_path() -> str:
+    """
+    Get the path to the dd_invoke.log file.
+    On Windows the default path is $Env:Temp\\dd_invoke.log
+    On Linux & MacOS the default path is /tmp/dd_invoke.log
+    """
+    if sys.platform == 'win32':
```

Comment:
You can use python's [tempfile.gettempdir()](https://python.readthedocs.io/en/latest/library/tempfile.html#tempfile.gettempdir)

Reply:
OMFG I was looking for this all day, thanks !

Reply:
After looking more into it, I don't think it solves my problem. The issue is that the file I'll write the logs in, needs to be monitored by the Log Agent. This means that in a `conf.yaml` file I'd need the path I'm putting here.

The `tempfile.gettempdir()` funtion has a lot of different outputs, so I think coding it this way is safer.

Reply:
> The issue is that the file I'll write the logs in, needs to be monitored by the Log Agent

What env is the Log Agent monitoring the file in ? If it's dev machines, then using the `TEMP` env var will still be an issue, as it is a per-user env var, so it will have different values when a dev runs the invoke task vs when the Logs Agent (`ddagentuser`) tries to locate the file.

If you need a consistent path across users, you should use `C:\Windows\Temp`, or `%WINDIR%\Temp` (could use [os.path.expandvars](https://docs.python.org/3/library/os.path.html#os.path.expandvars)

Reply:
Ok thanks ! I'll use `C:\Windows\Temp` as I need to put this path in the Log Agent config as well and it does not allow variable expansion.

Reply:
Fixed with [b31ce2a](https://github.com/DataDog/datadog-agent/pull/25540/commits/b31ce2aa01383ba0d41cfd27ad218ef241269ed8) ! Thanks Branden !